### PR TITLE
Retrieve and use api specific token when accessing tilavaraus-api

### PR DESCRIPTION
**Pull request by siren**

### CODE REVIEW

File: src/common/auth/LoggingIn.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <BrowserRouter>
+      <PageWrapper>
+        <LoadingSpinner />
+      </PageWrapper>
+    </BrowserRouter>
+  );
+};
+
+export default UpdateToken;
```
Comment (kieferbonk) [inline src/common/auth/LoggingIn.tsx:16]: I think you forgot to rename the component.

File: src/common/auth/axiosClient.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+  });
+
+  const { data } = response;
+
+  const apiAccessToken = data[apiScope];
+  setApiAccessToken(apiAccessToken, 'https://api.hel.fi/auth/tilavarausapidev');
```
Comment (kieferbonk) [inline src/common/auth/axiosClient.ts:50]: You probably want to configure the url per environment.

**Review by kieferbonk — CHANGES_REQUESTED** [review]

File: src/common/auth/LoggingIn.tsx (hunk 1 above)
Comment (siren) [reply src/common/auth/LoggingIn.tsx:16]: fixed

File: src/common/auth/axiosClient.ts (hunk 2 above)
Comment (siren) [reply src/common/auth/axiosClient.ts]: fixed

**Review by kieferbonk — APPROVED** [review]

**Pull request merged**